src/models/spørsmal.tsx

```diff
@@ -0,0 +1,23 @@
+export enum IKategori {
+  Medlemskap = 'medlemskap',
+  Barn = 'barn',
+}
+
+export enum Medlemskap {
```

Comment:
Hvordan skal denne brukes? :)

Reply:
@ninakjek , Medlemskap var ment for i bruken av IKategori, tror jeg :). Men hvis vi bestemmer oss for å ha spørsmålconfiger i hver sin fil, så vil denne være unødvendig og kan fjernes :) Samme med IKategori 😄